roles/openshift_docker_registry/vars/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+
```

Comment:
Just a nit, but would rather see us leave empty files out, since they aren't required for roles.